code/__HELPERS/pronouns.dm

```diff
@@ -1,259 +1,259 @@
-	. = "it"
-	if(capitalized)
-		. = capitalize(.)
+/datum/proc/p_they(temp_gender)
+	return "it"
 
-/datum/proc/p_their(capitalized, temp_gender)
-	. = "its"
-	if(capitalized)
-		. = capitalize(.)
+/datum/proc/p_They(temp_gender)
+	return capitalize(p_they(temp_gender))
+
+/datum/proc/p_their(temp_gender)
+	return "its"
+
+/datum/proc/p_Their(temp_gender)
+	return capitalize(p_their(temp_gender))
 
 /datum/proc/p_theirs(capitalized, temp_gender)
-	. = "its"
+	return "its"
 	if(capitalized)
-		. = capitalize(.)
+		return capitalize(.)
 
 /datum/proc/p_them(capitalized, temp_gender)
-	. = "it"
+	return "it"
 	if(capitalized)
-		. = capitalize(.)
+		return capitalize(.)
 
 /datum/proc/p_have(temp_gender)
-	. = "has"
+	return "has"
 
 /datum/proc/p_are(temp_gender)
-	. = "is"
+	return "is"
 
 /datum/proc/p_were(temp_gender)
-	. = "was"
+	return "was"
 
 /datum/proc/p_do(temp_gender)
-	. = "does"
+	return "does"
 
-/datum/proc/p_theyve(capitalized, temp_gender)
-	. = p_they(capitalized, temp_gender) + "'" + copytext_char(p_have(temp_gender), 3)
+/datum/proc/p_theyve(temp_gender)
+	return p_they(temp_gender) + "'" + copytext_char(p_have(temp_gender), 3)
 
-/datum/proc/p_theyre(capitalized, temp_gender)
-	. = p_they(capitalized, temp_gender) + "'" + copytext_char(p_are(temp_gender), 2)
+/datum/proc/p_Theyve(temp_gender)
+	return p_They(temp_gender) + "'" + copytext_char(p_have(temp_gender), 3)
+
+/datum/proc/p_theyre(temp_gender)
+	return p_they(temp_gender) + "'" + copytext_char(p_are(temp_gender), 2)
+
+/datum/proc/p_Theyre(temp_gender)
+	return p_They(temp_gender) + "'" + copytext_char(p_are(temp_gender), 2)
 
 /datum/proc/p_s(temp_gender) //is this a descriptive proc name, or what?
-	. = "s"
+	return "s"
 
 /datum/proc/p_es(temp_gender)
-	. = "es"
+	return "es"
 
 /datum/proc/plural_s(pluralize)
 	switch(copytext_char(pluralize, -2))
 		if ("ss")
-			. = "es"
+			return "es"
 		if ("sh")
-			. = "es"
+			return "es"
 		if ("ch")
-			. = "es"
+			return "es"
 		else
 			switch(copytext_char(pluralize, -1))
 				if("s", "x", "z")
-					. = "es"
+					return "es"
 				else
-					. = "s"
+					return "s"
 
 //like clients, which do have gender.
-/client/p_they(capitalized, temp_gender)
+/client/p_they(temp_gender)
 	if(!temp_gender)
 		temp_gender = gender
-	. = "they"
+	return "they"
 	switch(temp_gender)
 		if(FEMALE)
-			. = "she"
+			return "she"
 		if(MALE)
-			. = "he"
-	if(capitalized)
-		. = capitalize(.)
+			return "he"
 
-/client/p_their(capitalized, temp_gender)
+/client/p_their(temp_gender)
 	if(!temp_gender)
 		temp_gender = gender
-	. = "their"
+	return "their"
 	switch(temp_gender)
 		if(FEMALE)
-			. = "her"
+			return "her"
 		if(MALE)
-			. = "his"
-	if(capitalized)
-		. = capitalize(.)
+			return "his"
 
 /client/p_theirs(capitalized, temp_gender)
 	if(!temp_gender)
 		temp_gender = gender
-	. = "theirs"
+	return "theirs"
 	switch(temp_gender)
 		if(FEMALE)
-			. = "hers"
+			return "hers"
 		if(MALE)
-			. = "his"
+			return "his"
```

Comment:
regex strikes again (else case for the switch would work best I think, or just stick it below)

Reply:
Fixing it now!